ocrd_eval.schema.json

```diff
@@ -0,0 +1,238 @@
+        },
+        "eval_tool": {
+          "type": "string",
+          "description": "Human readable name and version of evaluation tool used (for UI"
+        },
+        "document_metadata": {
+          "type": "object",
+          "title": "Bibliographical and typographical metadata about the work to be evaluated",
+          "properties": {
+            "publication_year": {
+              "type": "number",
+              "description": "Year he document was originally published"
+            },
+            "publication_century": {
+              "type": "string",
+              "description": "Century he document was originally published",
+              "pattern": "[12][0-9]{3}-[12][0-9]{3}"
+            },
+            "publication_decade": {
+              "type": "string",
+              "description": "Decade the document was originally published",
+              "pattern": "[12][0-9]{3}-[12][0-9]{3}"
+            },
+            "number_of_pages": {
+              "type": "number",
+              "description": "Number of pages in this work (i.e. the number of images in the gt_workspace)"
+            },
+            "layout": {
+              "type": "string",
+              "enum": [
+                "simple",
+                "complex"
+              ]
+            },
+            "fonts": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "enum": [
+                  "antiqua",
+                  "fraktur"
+                ]
+              }
+            }
+          }
+        },
+        "provenance": {
+          "type": "object",
+          "description": "Information on which tools in which version were used in determining metrics",
+          "properties": {
+            "paramters": {
+              "type": "object",
+              "description": "Parameters passed to the evaluation processor"
+            }
+          }
+        }
+      }
+    },
+    "EvaluationReport": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "The metrics measured for this document",
+      "properties": {
+        "document_wide": {
+          "type": "object",
+          "description": "Document-wide metrics"
+        },
+        "by_page": {
+          "type": "array",
+          "description": "Metrics page-by-page",
+          "items": {
+            "type": "object",
+            "allOf": [
+              {
+                "properties": {
+                  "page_id": {
+                    "type": "string",
+                    "description": "PAGE ID"
+                  }
+                }
+              }
+            ]
+          }
+        }
+      }
+    },
+    "EvaluationMetrics": {
+      "cer": {
+        "description": "CER calculated over the text of a whole page (in by_page) or combined text of all pages (in document_wide)"
```

Comment:
If we look at `document_wide`: Is there a difference between `cer` and `cer_mean`?

Reply:
Same applies for `by_page`.

Reply:
IMO you have to drop the `cer` entry. It's `cer_mean` / `cer_median` / `cer_stddev` / `cer_range`, regardless of the hierarchy level.

Reply:
`cer` without qualifiers would be document-wide on the whole text, whereas the `_mean`, `_median` etc. suffixes indicate aggregate values over the pages.

But that's just my initial idea, if you think we don't need a global CER value for a whole-document, we can drop it.

Reply:
I don't understand. These two aspects (granularity and statistic) should be completely independent.

`EvaluationReport` has `document_wide` and `by_page`, each of which can use all of `EvaluationMetrics`.

What is `cer` other than `cer_mean`?

Reply:
I share @bertsky 's thoughts (that's what my initial question aims at).

Reply:
Ping @kba

Reply:
Agreed, let's drop `cer` without qualifier.

Reply:
30f1436